Framework/test/testCcdbDatabase.cxx

```diff
@@ -66,24 +65,20 @@ BOOST_AUTO_TEST_CASE(ccdb_create)
   f.backend->truncate("my/task", "*");
 }
 
-// this one sporadically fails when retrieving the list of objects under the "qc" directory
-/*
 BOOST_AUTO_TEST_CASE(ccdb_getobjects_name)
 {
   test_fixture f;
 
   CcdbDatabase* ccdb = static_cast<CcdbDatabase*>(f.backend.get());
   ILOG(Info) << "getListing()" << ENDM;
-  auto tasks = ccdb->getListing();
-  for (auto& task : tasks) {
-    ILOG(Info) << "getPublishedObjectNames of task " << task << ENDM;
-    auto objects = f.backend->getPublishedObjectNames(task);
-    for (auto& object : objects) {
-      Objects.insert({ task, object });
-    }
-  }
+  auto tasks = ccdb->getListing("/qc");
+  BOOST_CHECK_GT(tasks.size(), 5); // we know that there are a few
+  BOOST_CHECK_LT(tasks.size(), 100); // we know that there are not too many
+  // print but only for TST
+  auto objects = f.backend->getPublishedObjectNames("/qc/TST");
+  BOOST_CHECK_GT(objects.size(), 0);
+  BOOST_CHECK_LT(objects.size(), 200);
```

Comment:
I am afraid that someone eventually will break those limits be accident. Maybe it would be enough to check if size != 0?

Reply:
I agree